pkg/agent/baker.go

```diff
@@ -945,6 +948,15 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 	}
 }
 
+func getPortRangeEndValue(str, delimiter string, index int) int {
```

Comment:
can the port range ever be anything else than "A B"? if not then we probably don't need delimiter/index args here

Reply:
Good point, added it at first cause I tried to make it more generic, in the end it ended up like this :D

Reply:
maybe also worth trimming from the left and right side as well, though not sure if that would ever actually help